src/ansys/additive/core/additive.py

```diff
@@ -158,14 +196,25 @@ def _connect_to_servers(
 
         return connections
 
+    @property
+    def nsims_per_server(self) -> int:
+        """Number of simultaneous simulations to run on each server."""
+        return self._nsims_per_server
+
+    @nsims_per_server.setter
+    def nsims_per_server(self, value: int) -> None:
+        """Set the number of simultaneous simulations to run on each server."""
+        if value < 1:
+            raise ValueError("Number of simulations per server must be greater than zero")
```

Comment:
```suggestion
            raise ValueError("Number of simulations per server must be greater than zero.")
```
Message strings should have concluding punctuation.